android/src/main/java/com/swmansion/gesturehandler/core/GestureHandlerOrchestrator.kt

```diff
@@ -19,13 +20,10 @@ class GestureHandlerOrchestrator(
    * traversing view hierarchy and looking for gesture handlers.
    */
   var minimumAlphaForTraversal = DEFAULT_MIN_ALPHA_FOR_TRAVERSAL
-
-  private val gestureHandlers = arrayOfNulls<GestureHandler<*>?>(SIMULTANEOUS_GESTURE_HANDLER_LIMIT)
-  private val awaitingHandlers = arrayOfNulls<GestureHandler<*>?>(SIMULTANEOUS_GESTURE_HANDLER_LIMIT)
-  private val preparedHandlers = arrayOfNulls<GestureHandler<*>?>(SIMULTANEOUS_GESTURE_HANDLER_LIMIT)
-  private val handlersToCancel = arrayOfNulls<GestureHandler<*>?>(SIMULTANEOUS_GESTURE_HANDLER_LIMIT)
-  private var gestureHandlersCount = 0
-  private var awaitingHandlersCount = 0
+  private val gestureHandlers = arrayListOf<GestureHandler<*>?>()
+  private val awaitingHandlers = arrayListOf<GestureHandler<*>?>()
+  private val preparedHandlers = arrayListOf<GestureHandler<*>?>()
+  private val handlersToCancel = arrayListOf<GestureHandler<*>?>()
```

Comment:
You're right. I left `null` possibility because of how removing handler worked before last commit, but now I think it should be fine ([6957bb1](https://github.com/software-mansion/react-native-gesture-handler/pull/2672/commits/6957bb1387ac38cfad1c24e4c5b5e63333f7c28f))

Reply:
But I would be grateful if you could check if that approach is correct (line 74)

Reply:
I'm not sure what you mean, there's a closing bracket on line 74 😅